src/melee/ft/chara/ftMario/ftMario_SpecialHi.c

```diff
@@ -51,7 +51,7 @@ void ftMario_SpecialAirHi_Anim(HSD_GObj* gobj)
 }
 
 #define DEGREES_TO_RADIANS (3.14159265359f / 180.0f)
```

Comment:
Is this instance different bitwise from the define?
```suggestion
#define DEGREES_TO_RADIANS (3.14159265359f / 180.0f)
```

Reply:
Yeah not sure what to do with these, should probably consolidate all the deg-to-rad conversion constants as well. This is float-typed so it needs to be part of a macro or an inline.